build.jam

```diff
@@ -22,12 +22,13 @@ constant boost_dependencies :
     /boost/type_traits//boost_type_traits ;
 
 project /boost/property_tree
+    : common-requirements
```

Comment:
You shouldn't do this; the other way is the correct one, which is why we switched to it.